### PR TITLE
Fixes issues in which foreign key IDs were not displayed in the UI (+ a few others)

### PR DESCRIPTION
**Pull request by elijahbenizzy**
This is a bit ugly -- we're still identifying the underlying cause, but it turns out that in the SQL orm interaction with django ninja, foreign keys will not show up as IDs.

If you have foreign_key_id: int in the model it'll work. So adding this in fixed it.

### CODE REVIEW

File: ui/frontend/openapi-config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "schemaFile": "http://127.0.0.1:8000/api/openapi.json",
+  "schemaFile": "http://127.0.0.1:8241/api/openapi.json",
```
Comment (skrawcz) [inline ui/frontend/openapi-config.json:2]: just double checking, this is the right port?

File: ui/backend/server/trackingserver_template/api.py [hunk 2]
```diff
@@ -298,6 +298,10 @@ async def get_dag_template_catalog(
     node_templates = [
         NodeTemplateOut.from_orm(node_template) for node_template in all_node_templates
     ]
+    for i, node_template in enumerate(node_templates):
+        node_template.dag_template = all_node_templates[i].dag_template_id
+    # node_templates = []
+    # dag_template = node_template.dag_template_id
```
Comment (skrawcz) [inline ui/backend/server/trackingserver_template/api.py:304]: remove?

File: ui/backend/server/trackingserver_template/schema.py [hunk 3]
```diff
@@ -22,13 +22,20 @@ class Meta:
...
-        # exclude =
 
     primary_code_artifact: Optional[str] = None
 
+    # @classmethod
+    # def from_orm(cls, obj: Any, **kw: Any) -> "NodeTemplateOut":
+    #     out = super().from_orm(obj, **kw)
+    #     out.dag_template_id = obj.dag_template_id
+    #     return out
+
```
Comment (skrawcz) [inline ui/backend/server/trackingserver_template/schema.py:38]: remove?

**Review by skrawcz — APPROVED** [review]

File: ui/frontend/openapi-config.json (hunk 1 above)
Comment (elijahbenizzy) [reply ui/frontend/openapi-config.json:2]: yes

**Pull request merged**